definitions/infra-nginxserver/golden_metrics.yml

```diff
@@ -27,16 +27,30 @@ activeConnections:
       eventId: entityGuid
       eventName: entityName
 connectionsAccepted:
-  query:
-    eventId: entityGuid
-    select: average(`net.connectionsAcceptedPerSecond`)
-    from: NginxSample
+  queries:
+    newRelic:
+      select: average(nginx.server.net.connectionsAcceptedPerSecond)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name
```

Comment:
Given that you're using the default values here, I don't think it's useful to set these values explicitly

Reply:
What I'm saying is that, as it's stated [here](https://github.com/newrelic/entity-definitions/blob/main/docs/golden_metrics.md#query-options) there are default values for the query/queries configuration objects. 
The default value for `from` is `Metric`, for `eventId` is `entity.guid` and for `eventName` is `entity.name`. 
For this reason, you don't need to override them on lines 33-35.